packs/data/agents-of-edgewatch-bestiary.db/almiraj.json

```
@@ -202,7 +202,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>Any living creature slain by the almiraj's horn Strike immediately becomes @Compendium[pf2e.conditionitems.Petrified]{Petrified} with no saving throw, in addition to dying.</p>\n<p>A severed almiraj horn retains a wisp of its former magic; in combat, the horn can be wielded as a <em>@Compendium[pf2e.equipment-srd.Dagger]{+1 dagger}</em> or, if affixed to a shaft, a +1 <em>@Compendium[pf2e.equipment-srd.Spear]{spear}</em></p>"
+                    "value": "<p>Any living creature slain by the almiraj's horn Strike immediately becomes @Compendium[pf2e.conditionitems.Petrified]{Petrified} with no saving throw, in addition to dying.</p>\n<p>A severed almiraj horn retains a wisp of its former magic; in combat, the horn can be wielded as a <em>@Compendium[pf2e.equipment-srd.Dagger]{+1 dagger}</em> or, if affixed to a shaft, a <em>@Compendium[pf2e.equipment-srd.Spear]{+1 spear}</em></p>"
```

```suggestion
                    "value": "<p>Any living creature slain by the almiraj's horn Strike immediately becomes @Compendium[pf2e.conditionitems.Petrified]{Petrified} with no saving throw, in addition to dying.</p>\n<p>A severed almiraj horn retains a wisp of its former magic; in combat, the horn can be wielded as a <em>+1 @Compendium[pf2e.equipment-srd.Dagger]{dagger}</em> or, if affixed to a shaft, a <em>+1 @Compendium[pf2e.equipment-srd.Spear]{spear}</em></p>"
```
I think with this one I wouldn't want it to imply that the dragged in item would have the +1.